Core/HLE/sceNet.cpp

```diff
@@ -1219,21 +1226,57 @@ static int sceNetApctlGetBSSDescEntryUser(int entryId, int infoId, u32 resultAdd
 	if (!Memory::IsValidAddress(resultAddr))
 		hleLogError(SCENET, -1, "apctl invalid arg");
 
+	// Generate an SSID name
+	char dummySSID[APCTL_SSID_MAXLEN] = "WifiAP0";
+	dummySSID[6] += static_cast<char>(entryId);
+
 	switch (infoId) {
 	case PSP_NET_APCTL_DESC_IBSS: // IBSS, 6 bytes
-		Memory::WriteStruct(resultAddr, &netApctlInfo.bssid);
+		if (entryId == 0)
+			Memory::WriteStruct(resultAddr, &netApctlInfo.bssid);
+		else {
+			// Generate a BSSID/MAC address
+			char dummyMAC[ETHER_ADDR_LEN];
+			memset(&dummyMAC, entryId, sizeof(dummyMAC));
```

Comment:
Nitpick, but `char dummyMAC[ETHER_ADDR_LEN] {};` will yield the same result without an explicit memset.

Reply:
i see, i thought initializing it with {} have the same effect as {0} which fill it with 0 and without it became uninitialized, well we don't really need to null-terminating a mac address anyway since it's not a string, so i should probably remove it.

Reply:
I think CookiePLMonster thought you were zero-initializing it, in which case it would have worked.

You can remove the & sign though, leaving `memset(dummyMAC, entryId, sizeof(dummyMAC));`

Reply:
ah right, if i wanted to use `&` i need to explicitly pointing it to the first char like `&dummy[0]`